deploy/resources/devel/pipelines/00-release.yaml

```diff
@@ -1722,6 +1722,8 @@ kind: RoleBinding
 metadata:
   name: openshift-pipelines-prometheus-k8s-read-binding
   namespace: tekton-pipelines
+  annotations:
+    operator.tekton.dev/preserve-rb-subject-namespace: "true"
```

Comment:
let us avoid editing the pipeline release.yaml.

Instead of filtering and conditionally changing the subject namespace, could we use better filtering.
may be using manifestival provides ByName. In addition, can we write a custom `predicate` if required.

Reply:
Actually, this pipeline.yaml would come from https://github.com/openshift/tektoncd-pipeline/pull/485 here. So we aren't editing it but synchronising it with what we should have.

Reply:
oh, i see..  cool :+1: